src/control/server/mgmt_drpc.go

```diff
@@ -23,7 +23,7 @@
 
 package server
 
-// #cgo CFLAGS: -I${SRCDIR}/../../../include
```

Comment:
Was this path change intentional? I really hate these relative includes, but until we fix the bigger build issues I guess we're stuck with them.

Reply:
It was intentional. I bumped into include failures when running the run_go_tests.sh script on its own. I guess the file must have moved at some point and this wasn't fixed.